### ISSUE TITLE
Travis CI integration

### ISSUE THREAD

**Issue by jonathanverner** [brython-dev/brython#496]
It would be nice if we could run the tests on Travis CI. This probably worked in the past, but now is defunct (see also Issue #491)

**Comment by jonathanverner** [comment]
I will try to work on this issue in my **ci** branch https://github.com/jonathanverner/brython/tree/ci.
RIght now the branch contains just an updated `.travis.yml` (paths have changed) and the `run_test.html` loads the `unicode.min.js`. This makes the test suite run in chromium (with one failure in the `Issues` module). Unfortunately, PhantomJS chokes on it somewhere and does not even start running the tests :-(

**Comment by olemis** [comment]
Great ! Nonetheless , for the sake of being consistent let's follow this discussion and development in #234 . You might also want to take a look at [my private travis-ci branch]() and merge it in your local working copy .

BTW, I use to reference ticket number in commit messages for changesets to be shown in ticket history .

/me closing this issue as duplicate .

**Comment by jonathanverner** [comment]
Sorry, I should have looked before starting a new issue...
